dist/ivh-treeview.js

```diff
@@ -108,6 +108,36 @@ angular.module('ivh.treeview').directive('ivhTreeviewChildren', function() {
 
 
 /**
+ * Click logic for treeview nodes
+ *
+ * Handles node click on element click
+ *
+ * @private
+ * @package ivh.treeview
+ * @copyright 2015 NorthPage, Inc.

```

Comment:
I don't think copyrights exist in open source: http://www.ifosslr.org/ifosslr/article/view/30